### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by mariogiov**
Double negatives don't work

### CODE REVIEW

File: ngi_pipeline/conductor/flowcell.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def process_demultiplexed_flowcells(demux_fcid_dirs, restrict_to_projects=None,
         projects_to_analyze = projects_to_analyze.values()
     for project in projects_to_analyze:
         project_status = CharonSession().project_get(project.project_id)['status']
-        if not project_status != "OPEN":
+        if not project_status == "OPEN":
```
Comment (guillermo-carrasco) [inline ngi_pipeline/conductor/flowcell.py]: outch

Comment (mariogiov) [inline ngi_pipeline/conductor/flowcell.py]: YeahI honestly don't know which way BDFL wants me to write this. Should I still be using `not` or just go with `!=`? In any event I should probably not use both. Although that would be a funny way to test for truth.

Comment (remiolsen) [inline ngi_pipeline/conductor/flowcell.py]: `project_status is not "OPEN"` ?

Comment (mariogiov) [inline ngi_pipeline/conductor/flowcell.py]: Ooh yeah I think that's a winner. Good call.

**Pull request merged**